src/services/EcoBridge/Lifi/Lifi.selectors.ts

```diff
@@ -0,0 +1,152 @@
+  createSelector([(state: AppState) => state.ecoBridge[bridgeId].txBridgingData], txBridgingData => txBridgingData)
+
+const createSelectOwnedTransactions = (bridgeId: LifiList) =>
+  createSelector(
+    [
+      (state: AppState) => state.ecoBridge[bridgeId].transactions,
+      (_state: AppState, account: string | undefined) => account,
+    ],
+    (txs, account) => {
+      let ownedTxs: LifiTransactionStatus[] = []
+      if (account) {
+        ownedTxs = txs.reduce<LifiTransactionStatus[]>((filteredTxs, tx) => {
+          if (account.toLowerCase() === tx.account.toLowerCase()) {
+            filteredTxs.push(tx)
+          }
+
+          return filteredTxs
+        }, [])
+      }
+
+      return ownedTxs
+    }
+  )
+
+const createSelectPendingTransactions = (selectOwnedTxs: ReturnType<typeof createSelectOwnedTransactions>) =>
+  createSelector(selectOwnedTxs, ownedTxs => {
+    const pendingTxs = ownedTxs.filter(
+      ({ statusResponse }) => statusResponse.status !== 'FAILED' && !statusResponse.receiving?.txHash
+    )
+
+    return pendingTxs
+  })
+
+const createSelectFailedTransactions = (selectOwnedTxs: ReturnType<typeof createSelectOwnedTransactions>) =>
+  createSelector(selectOwnedTxs, ownedTxs => {
+    const failedTxs = ownedTxs.filter(
+      ({ statusResponse }) => statusResponse.status === 'FAILED' && !statusResponse.receiving?.txHash
+    )
+
+    return failedTxs
+  })
+
+const createSelectBridgeTransactionsSummary = (
+  bridgeId: LifiList,
+  selectOwnedTxs: ReturnType<typeof createSelectOwnedTransactions>
+) =>
+  createSelector([selectOwnedTxs], txs => {
+    const summaries = txs.map(({ statusResponse, timeResolved }) => {
+      const pendingReason =
+        statusResponse.status === LifiTxStatus.PENDING && statusResponse.substatus
+          ? LIFI_PENDING_REASONS[statusResponse.substatus]!
+          : 'We cannot determine the status of the transfer.'
+
+      const fromValue = Number(statusResponse.sending.amount).toString()
+      const toValue = Number(statusResponse.receiving?.amount ?? 0).toString()
+
+      const summary: BridgeTransactionSummary = {
+        assetName: statusResponse.sending.token?.name!,
+        assetAddressL1: statusResponse.sending.token?.address,
+        assetAddressL2: statusResponse.receiving?.token?.address,
+        // @ts-ignore ChainId type missmatch.'
+        fromChainId: statusResponse.sending.chainId!,
+        // @ts-ignore ChainId type missmatch.'
+        toChainId: statusResponse.receiving?.chainId!,
+        status:
+          statusResponse.status === LifiTxStatus.DONE
+            ? BridgeTransactionStatus.CONFIRMED
+            : statusResponse.status === LifiTxStatus.FAILED || statusResponse.status === LifiTxStatus.INVALID
+            ? BridgeTransactionStatus.FAILED
+            : BridgeTransactionStatus.PENDING,
+        fromValue,
+        toValue,
+        txHash: statusResponse.receiving?.txHash!,
+        pendingReason,
+        // TODO: need to fix this
```

Comment:
I gave `Date.now()` when DONE but that is should work for now since we are not fetching history. I will add comment to fetch the correct time from onChian